.github/scripts/docker/ScalaCliSlimDockerFile

```diff
@@ -1,3 +1,3 @@
-FROM gcr.io/distroless/base-debian10
+FROM gcr.io/distroless/java-debian10
```

Comment:
I'm not sure we should switch to `gcr.io/distroless/java-debian10`. It weights much more than `base-debian10` (~200MB vs ~40MB). Couldn't [this trick](https://github.com/GoogleContainerTools/distroless/issues/217#issuecomment-404551174) work to add libz in the image?

Reply:
I'm not sure. For some reason, the java-debian10 image contains much more libraries than just zlib. I don't really know why there are so many, but I guess Java loads some of them at runtime. 

```
~/slim-java$ docker run --entrypoint sh -ti gcr.io/distroless/base-debian10:debug 
/ # ls /usr/lib/x86_64-linux-gnu/
audit             engines-1.1       gconv             libcrypto.so.1.1  libssl.so.1.1
~/slim-java$ docker run --entrypoint sh -ti gcr.io/distroless/java-debian10:debug 
/ # ls /usr/lib/x86_64-linux-gnu/
audit                       libexpatw.so.1              libgio-2.0.so.0             libgobject-2.0.so.0         libgraphite2.so.3.2.1       libjpeg.so.62.2.0           libpng16.so.16.36.0
engines-1.1                 libexpatw.so.1.6.8          libgio-2.0.so.0.5800.3      libgobject-2.0.so.0.5800.3  libgthread-2.0.so.0         liblcms2.so.2               libssl.so.1.1
gconv                       libfontconfig.so.1          libglib-2.0.so.0            libgomp.so.1                libgthread-2.0.so.0.5800.3  liblcms2.so.2.0.8           libstdc++.so.6
gio                         libfontconfig.so.1.12.0     libglib-2.0.so.0.5800.3     libgomp.so.1.0.0            libharfbuzz.so.0            libpcreposix.so.3           libstdc++.so.6.0.25
glib-2.0                    libfreetype.so.6            libgmodule-2.0.so.0         libgraphite2.so.2.0.0       libharfbuzz.so.0.20301.0    libpcreposix.so.3.13.3
libcrypto.so.1.1            libfreetype.so.6.16.1       libgmodule-2.0.so.0.5800.3  libgraphite2.so.3           libjpeg.so.62               libpng16.so.16
```